src/components/SheetBuilder/SheetBuilderForm/SheetBuilderFormStep/SheetBuilderFormStepAttributesDefinition/AttributeInputButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ComponentProps, PropsWithChildren } from 'react';
+
+type Props = {
+  onClick: () => void;
+  side: 'left' | 'right';
+};
+
+const AttributeInputButton = ({
+  onClick,
+  children,
+  side,
+}: PropsWithChildren<Props>) => {
+  let className: ComponentProps<'div'>['className'] = '';
+
+  if (side === 'left') {
+    className = 'rounded-tl-2xl rounded-bl-2xl';
+  }
+
+  if (side === 'right') {
+    className = 'rounded-tr-2xl rounded-br-2xl';
+  }
```

Comment:
Se não existe a possibilidade das classes serem vazias (já que o side sempre vai ser `left` ou `right`), são poucas opções e tem suas classes bem definidas, vale a pena definir um objeto e acessar isso dinamicamente.

```suggestion
  const sidesClasses = {
    right: 'rounded-tl-2xl rounded-bl-2xl',
    left:  'rounded-tr-2xl rounded-br-2xl'  
  }
  
```

dai no template, voce substituir `className` por `sidesClasses[side]`

Reply:
indeed